python/paddle/trainer_config_helpers/layers.py

```diff
@@ -5477,7 +5477,9 @@ def nce_layer(input,
                              A uniform distribution will be used if not provided.
                              If not None, its length must be equal to num_classes.
     :type neg_distribution: list|tuple|collections.Sequence|None
-    :param bias_attr: Bias parameter attribute. True if no bias.
+    :param bias_attr: The Bias Attribute. If no bias, then pass False or
+                      something not type of ParameterAttribute. None will get a
+                      default Bias.
```

Comment:
- If this parameter is set to False, no bias is defined. If this parameter is set to None, bias with default initialization settings is defined. 
- please help to check what is the default setting of the bias parameter and add it into the doc,

Reply:
Done

Reply:
please fix this.